### PR TITLE
add datetime and peers's network addr to diagnostics

### PR DESCRIPTION
**Pull request by biluohc**
Add peers' network addresses to the diagnostics API.

### CODE REVIEW

**Comment by pdyraga** [comment]
Thanks for your contribution! 🙌  We need to take a closer look, it's possible we have a duplicate with https://github.com/keep-network/keep-core/pull/2074

**Comment by biluohc** [comment]
Ok, i will let ci pass tomorrow

File: pkg/diagnostics/diagnostics.go [hunk 1]
```diff
@@ -78,6 +89,8 @@ func RegisterClientInfoSource(
 		clientInfo := map[string]interface{}{
 			"network_id":       clientID,
 			"ethereum_address": key.NetworkPubKeyToEthAddress(clientPublicKey),
+			"network_addrs":    connectionManager.NetAddrStrings(),
+			"datetime":         time.Now().Format("2006-01-02 15:04:05"),
```
Comment (Shadowfiend) [inline pkg/diagnostics/diagnostics.go:93]: Circling back here after a long time, I realize---what's the reasoning for including a timestamp in the JSON here? I believe Go's default HTTP response includes a `Date` header that should make this redundant.

Comment (biluohc) [reply pkg/diagnostics/diagnostics.go:93]: Thank you for your reminder, I mistakenly thought that Diagnostics also has a tick like Metrics.

But the timestamp in JSON may be more convenient than the HTTP header, and there is only a little overhead. I can also delete it if you want to delete it.

Is there any other problem with the code, let me update code together

**Pull request closed without merging**